bower.json

```diff
@@ -12,7 +12,7 @@
   "description": "jQuery Split Pane plugin",
   "license": "MIT",
   "dependencies": {
-    "jquery": "1.11.1"
+    "jquery": ">=1.11.1"
```

Comment:
I have updated bower dep to "~1.11.1". Now in version 0.5.1. If you want jquery 2 I suggest creating a branch. My plugin is supporting IE 8. By allowing >= 1.1.1 i would also allow major changes that might brake my plugin.